src/DssSpell.t.base.sol

```diff
@@ -929,6 +985,110 @@ contract DssSpellTestBase is Config, DSTest, DSMath {
         assertEq(token.balanceOf(address(join)), 0);
     }
 
+    function checkCropCRVLPIntegration(
+        bytes32 _ilk,
+        CropJoinLike join,
+        ClipAbstract clip,
+        CurveLPOsmLike pip,
+        address _medianizer1,
+        address _medianizer2,
+        bool _isMedian1,
+        bool _isMedian2,
+        bool _checkLiquidations
+    ) public {
+        pip.poke();
+        hevm.warp(block.timestamp + 3601);
+        pip.poke();
+        spotter.poke(_ilk);
+
+        // Check medianizer sources
+        assertEq(pip.orbs(0), _medianizer1);
+        assertEq(pip.orbs(1), _medianizer2);
+
+        // Authorization
+        assertEq(join.wards(pauseProxy), 1);
+        assertEq(vat.wards(address(join)), 1);
+        assertEq(clip.wards(address(end)), 1);
```

Comment:
I think the following auth checks are missing and should be included to ensure good coverage:
```suggestion
        assertEq(vat.wards(address(clip)), 1);
        assertEq(dog.wards(address(clip)), 1);
        assertEq(clip.wards(address(dog)), 1);
        assertEq(clip.wards(address(end)), 1);
        assertEq(clip.wards(address(esm)), 1);
```

Reply:
added

Reply:
I think there is a redundant clip wards end check in L1011, already present in L1015